factory/__init__.py

```diff
@@ -51,6 +52,12 @@
     stub_batch,
 )
 
+for optmod in ["alchemy", "django", "mogo", "mongoengine"]:
+    try:
+        importlib.import_module(".", f"{__package__}.{optmod}")
+    except ImportError:
+        pass
+
```

Comment:
Can't we do the same with a little less magic?

```python
try:
    import alchemy
except ImportError:
    pass
try:
    import django
except ImportError:
    pass
try:
    import mogo
except ImportError:
    pass
try:
    import mongoengine
except ImportError:
    pass

```

adds 17 lines instead of the 7 in this PR, but is much easier to read and grep for ;)